### ISSUE TITLE
JS conflict with other Blocks Plugins. Related to @wordpress/scripts - webpack runtime issue

### ISSUE THREAD

**Issue by CP-Antoine** [liip/bootstrap-blocks-wordpress-plugin#64]
Hi Liip team,

First, thanks a lot for your plugin.

I had JS conflict issue with your plugin and another one (that I coded based on` @wordpress/create-block` package).
```
Your site doesn't includes support for the "wp-bootstrap-blocks/container".
You can leave this block intact, convert its content to a Custom HTML block, or remove it entirely.
```
**To reproduce**

1. Using WordPress 5.5.3, with TwentyTwenty.
2. Install and activate the `bootstrap-blocks-wordpress-plugin`, as the only activated plugin.
3. Add a `wp-bootstrap-blocks/container` block and Save
4. Generate the out of the box `esnext-example` plugin using `npx @wordpress/create-block`
5. Do `npm run build` on this plugin
6. Install and activate this plugin
7. Try to edit the page where block was added previously

I manage to get all blocks plugin to work together with the workaround described here:
https://github.com/WordPress/gutenberg/issues/23607#issuecomment-656557883

**Comment by tschortsch** [comment]
Thanks a lot for your detailed issue. I‘ll try to look into it as soon as possible. We already had a very similar issue: https://github.com/liip/bootstrap-blocks-wordpress-plugin/issues/48. I‘ll come back to you as soon as I have more information.

**Comment by tschortsch** [comment]
I could finally look into this issue. It will be fixed in the next release of the plugin (v3.1.1).

**Comment by tschortsch** [comment]
We‘ve just released v3.1.1 of the plugin which fixes the described bug.
